spec/GraphQL.md

```diff
@@ -4,125 +4,37 @@ _Current Working Draft_
 
 **Introduction**
 
-This is the specification for GraphQL, a query language and execution engine
-originally created at Facebook in 2012 for describing the capabilities and
-requirements of data models for client-server applications. The development of
-this open standard started in 2015. This specification was licensed under OWFa
-1.0 in 2017. The [GraphQL Foundation](https://graphql.org/foundation/) was
-formed in 2019 as a neutral focal point for organizations who support the
-GraphQL ecosystem, and the
+This is the specification for GraphQL, a query language and execution engine for
+describing and performing the capabilities and requirements of data models for
+client-server applications.
+
+GraphQL was originally created in 2012 and the development of this open standard
+started in 2015. This specification was licensed under OWFa 1.0 in 2017 (see
+[Licensing](#sec-Copyright-and-Licensing)). The
+[GraphQL Foundation](https://graphql.org/foundation/) was formed in 2019 as a
+neutral focal point for organizations who support the GraphQL ecosystem, and the
 [GraphQL Specification Project](https://graphql.org/community/) was established
 also in 2019 as the Joint Development Foundation Projects, LLC, GraphQL Series.
-
 If your organization benefits from GraphQL, please consider
 [becoming a member](https://graphql.org/foundation/join/#graphql-foundation) and
 helping us to sustain the activities that support the health of our neutral
 ecosystem.
 
-The GraphQL Specification Project has evolved and may continue to evolve in
-future editions of this specification. Previous editions of the GraphQL
-specification can be found at permalinks that match their
+This specification is developed on GitHub at
+[graphql/graphql-spec](https://github.com/graphql/graphql-spec/). Contributions
+are managed by the
+[GraphQL Working Group](https://github.com/graphql/graphql-wg), hosted by the
+[GraphQL TSC](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md).
```

Comment:
We shouldn't use an abbreviation the first time the term is introduced:

```suggestion
[GraphQL Technical Steering Committee](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md).
```

(Feel free to append ` (TSC)` if we want to refer to it again, but I don't think we do.)